src/rust/cryptography-x509-verification/src/types.rs

```diff
@@ -138,6 +138,13 @@ impl<'a> DNSPattern<'a> {
             },
         }
     }
+
+    pub fn inner_name(&self) -> &DNSName<'a> {
```

Comment:
Can you add a comment describing this behavior?

Reply:
Whoops, definitely.

Reply:
7b32b006e..c95a21efa